server/utils/nunjucksSetup.ts

```diff
@@ -2,6 +2,10 @@
 import nunjucks from 'nunjucks'
 import express from 'express'
 import * as pathModule from 'path'
+import { PrisonApiOffenderSentenceAndOffences } from '../@types/prisonApi/prisonClientTypes'
+
```

Comment:
Seems a little "too coupled" to have prisonAPI types getting into the nunjucks filters? Could it be separated such that a view model is used here? Just a thought.